example/run_demo.py

```diff
@@ -29,15 +30,21 @@
     KaiRpcApplicationConfig,
 )
 
+
+def get_binary_path(path: str) -> Path:
+    if platform.system().lower() != 'windows' and ".exe" not in path.lower():
```

Comment:
I think you need to change this to` == 'windows'` because it is adding .exe for ubuntu.

Reply:
Oops, PR of one line and I got it wrong :'(  I left it while I was testing, solved, thanks!